.github/workflows/CI.yml

```diff
@@ -3,36 +3,44 @@ name: CI
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
Update the main branch to trigger workflow.

Reply:
Looks good - havent checked on this in a while